screens/VerifyIdentityOverlay.tsx

```diff
@@ -8,11 +8,21 @@ import {VC} from '../types/VC/ExistingMosipVC/vc';
 import {Modal} from '../components/ui/Modal';
 import {t} from 'i18next';
 import {useTranslation} from 'react-i18next';
+import {VCMetadata} from '../shared/VCMetadata';
 
 export const VerifyIdentityOverlay: React.FC<
   VerifyIdentityOverlayProps
 > = props => {
   const {t} = useTranslation('VerifyIdentityOverlay');
+  const isOpenId4VCI =
+    props.vc?.vcMetadata &&
+    VCMetadata.fromVC(props.vc?.vcMetadata).isFromOpenId4VCI();
+  const credential = isOpenId4VCI
+    ? props.vc?.verifiableCredential
+    : props.vc?.credential;
+  const vcImage = isOpenId4VCI
+    ? props.vc?.verifiableCredential.credential.credentialSubject.face
+    : props.vc?.credential?.biometrics.face;
   return (
```

Comment:
can we simplify this for readability?
const isOpenId4VCI =
    props.vc?.vcMetadata &&
    VCMetadata.fromVC(props.vc?.vcMetadata).isFromOpenId4VCI();
let credential;
let vcImage;
if (isOpenId4VCI) {
  credential = props.vc?.verifiableCredential;
  vcImage = props.vc?.verifiableCredential.credential.credentialSubject.face;
} else {
  credential = props.vc?.credential;
  vcImage = props.vc?.credential?.biometrics.face;
}

Reply:
**"ternary operator is faster than if-else"** and better code